Add link to viewer page from owner doorboard

Burge337 opened the pull request:
I kinda want to change the wording of the button to something like "student view" or "viewer perspective" but it's easy enough to change later on if we don't like "view doorboard".
Closes issue #12

Burge337 committed:
Add link to viewer page from owner doorboard

Closes issue #12

helloworld12321 reviewed (approved):
Super cool, looks good! :slightly_smiling_face:

Re: wording, that's an interesting question! That sounds like a thing we can maybe ask Profs. KK & Nic about on Monday :)

helloworld12321 committed:
Merge branch 'staging' into add-link-to-viewer-page

helloworld12321 committed:
Clean up after merging

(Since we merged in staging, OwnerComponent.owner is now an observable, so we need to update the HTML accordingly.)
